### PR TITLE
feat: Avatar.Group support size props

### PR DESCRIPTION
**Pull request by zhouyashu279**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [X] New feature
- [ ] Bug fix
- [X] Site / documentation update
- [X] Demo update
- [X] Component style update
- [X] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [X] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |        add size prop for Avatar.Group   |
| 🇨🇳 Chinese |       Avatar.Group组件增加size属性    |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/avatar/demo/group.md](https://github.com/zdaizi/ant-design/blob/feature/components/avatar/demo/group.md)
[View rendered components/avatar/index.en-US.md](https://github.com/zdaizi/ant-design/blob/feature/components/avatar/index.en-US.md)
[View rendered components/avatar/index.zh-CN.md](https://github.com/zdaizi/ant-design/blob/feature/components/avatar/index.zh-CN.md)

### CODE REVIEW

**Comment by hengkx** [comment]
please check 
image

File: components/avatar/demo/group.md [hunk 1]
```diff
@@ -30,13 +30,26 @@ class Demo extends React.Component {
...
-        <Avatar.Group maxCount={2} maxStyle={{ color: '#f56a00', backgroundColor: '#fde3cf' }}>
-          <Avatar src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png" />
-          <Avatar style={{ backgroundColor: '#f56a00' }}>K</Avatar>
+        <Avatar.Group
+          maxCount={2}
+          maxStyle={{ color: '#f56a00', backgroundColor: '#fde3cf' }}
+          size="small"
+        >
+          <Avatar
+            size="small"
```
Comment (hengkx) [inline components/avatar/demo/group.md:39]: Group has set the size, why do you need to set it here?

File: components/avatar/index.en-US.md [hunk 2]
```diff
@@ -26,8 +26,9 @@ Avatars can be used to represent people or objects. It supports images, `Icon`s,
...
-| ------------------- | -------------------------------------- | ----------------- | ------ | ---- |
-| maxCount            | Max avatars to show                    | number            | -      |      |
-| maxStyle            | The style of excess avatar style       | CSSProperties     | -      |      |
-| maxPopoverPlacement | The placement of excess avatar Popover | `top` \| `bottom` | `top`  |      |
+| 参数 | 说明 | 类型 | 默认值 | 版本 |
+| --- | --- | --- | --- | --- |
+| maxCount | Max avatars to show | number | - |  |
+| maxStyle | The style of excess avatar style | CSSProperties | - |  |
+| maxPopoverPlacement | The placement of excess avatar Popover | `top` \| `bottom` | `top` |  |
+| size | The size of excess avatar | number \| `large` \| `small` \| `default` \| `{ xs: number, sm: number, ...}` | `default` |  |
```
Comment (hengkx) [inline components/avatar/index.en-US.md:34]: version 4.8.0

**Comment by hengkx** [comment]
#27068 Size should work for all avatars

File: components/avatar/demo/group.md (hunk 1 above)
Comment (zdaizi) [reply components/avatar/demo/group.md:39]: 抱歉  我来改下

Comment (zdaizi) [reply components/avatar/demo/group.md:39]: 您好， 请问我应该close这pr, 更改完之后重新pr吗？

Comment (hengkx) [reply components/avatar/demo/group.md:39]: 可以直接改，为什么要重新一个

Comment (zdaizi) [reply components/avatar/demo/group.md]: 请问在哪里改？

File: components/avatar/__tests__/__snapshots__/Avatar.test.js.snap [hunk 3]
```diff
@@ -94,6 +94,43 @@ exports[`Avatar Render fallback 1`] = `
 </span>
 `;
 
+exports[`Avatar Render group support size props 1`] = `
+<div
+  class="ant-avatar-group"
+>
+  <span
+    class="ant-avatar ant-avatar-circle"
+  >
```
Comment (hengkx) [inline components/avatar/__tests__/__snapshots__/Avatar.test.js.snap:103]: size属性需要传递给 avatar,你写的size为small 这边并没有small 的样式

Comment (hengkx) [reply components/avatar/__tests__/__snapshots__/Avatar.test.js.snap:103]: image
家里备用电脑有点问题  将就看下

File: components/avatar/index.en-US.md [hunk 4]
```diff
@@ -26,8 +26,9 @@ Avatars can be used to represent people or objects. It supports images, `Icon`s,
...
-| ------------------- | -------------------------------------- | ----------------- | ------ | ---- |
-| maxCount            | Max avatars to show                    | number            | -      |      |
-| maxStyle            | The style of excess avatar style       | CSSProperties     | -      |      |
-| maxPopoverPlacement | The placement of excess avatar Popover | `top` \| `bottom` | `top`  |      |
+| 参数 | 说明 | 类型 | 默认值 | 版本 |
+| --- | --- | --- | --- | --- |
+| maxCount | Max avatars to show | number | - |  |
+| maxStyle | The style of excess avatar style | CSSProperties | - |  |
+| maxPopoverPlacement | The placement of excess avatar Popover | `top` \| `bottom` | `top` |  |
+| size | The size of excess avatar | number \| `large` \| `small` \| `default` \| `{ xs: number, sm: number, ...}` | `default` | 4.8.0 |
```
Comment (hengkx) [inline components/avatar/index.en-US.md:34]: @afc163  `{ xs: number, sm: number, ...}` 这个属性 需要在下面单独写完整 这边引用吗？

File: components/avatar/demo/group.md (hunk 1 above)
Comment (hengkx) [reply components/avatar/demo/group.md]: 你要改什么？

Comment (hengkx) [reply components/avatar/demo/group.md]: 你不是已经删掉了

File: components/avatar/__tests__/__snapshots__/Avatar.test.js.snap (hunk 3 above)
Comment (zdaizi) [reply components/avatar/__tests__/__snapshots__/Avatar.test.js.snap:103]: avatar.group上的size=small,  加到 多余头像class上了：<span class="ant-avatar-sm">

File: components/avatar/index.en-US.md (hunk 4 above)
Comment (afc163) [reply components/avatar/index.en-US.md:34]: 写完整吧。

File: components/avatar/__tests__/__snapshots__/Avatar.test.js.snap (hunk 3 above)
Comment (zdaizi) [reply components/avatar/__tests__/__snapshots__/Avatar.test.js.snap:103]: <avatar.group>的size="small", 加到多余头像标签上了：<span class="ant-avatar-sm">

Comment (zdaizi) [reply components/avatar/__tests__/__snapshots__/Avatar.test.js.snap:103]: 老哥， 这现在要咋改啊？

**Comment by hengkx** [comment]
ping

**Comment by zdaizi** [comment]
> ping


> ping

？？

**Comment by zdaizi** [comment]
> ping

？？

**Comment by hengkx** [comment]
> > ping
> 
> 
> 
> ？？

亲 这个pr 还需要 完善。 ci 也还没有通过

**Comment by hengkx** [comment]
我这边 要用 这个 属性了，我重新 开一个 pr了🙏 你可以参考 一下

**Pull request closed without merging**